study1/week1/set-mismatch.java

```diff
@@ -0,0 +1,27 @@
+class Solution {
+    public int[] findErrorNums(int[] nums) {
+        //time complexity: O(n)
+        //space complexiti: O(n) -> because of hashSet..?
+        int n = nums.length;
+        int[] resultArr = new int[2];
+        int expectSum = 0;
+        int actualSum = 0;
+
+        HashSet<Integer> numsSet = new HashSet<Integer>();
+
+        for(int i = 0; i < n; i++) {
```

Comment:
Nit: you need a space between `for` and `{`
https://google.github.io/styleguide/javaguide.html#s4.6.2-horizontal-whitespace